### PR TITLE
TW-816: [Mobile] change the UI for create new group chat

### PR DESCRIPTION
**Pull request by sherlockvn**
### Demo mobile:

https://github.com/linagora/twake-on-matrix/assets/43041967/64fc648d-df07-4793-8f3b-e98babd1442b

### CODE REVIEW

**Review by drminh2807 — APPROVED** [review]

**Review by drminh2807 — APPROVED** [review]

**Review by drminh2807 — APPROVED** [review]

File: lib/pages/new_group/new_group_chat_info.dart [hunk 1]
```diff
@@ -320,3 +330,100 @@ class _AvatarForWebBuilder extends StatelessWidget {
...
+
+  const _EncryptionSettingTile({
+    required this.enableEncryptionNotifier,
+    this.onChanged,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 8.0, bottom: 8.0, right: 4.0),
```
Comment (nqhhdev) [inline lib/pages/new_group/new_group_chat_info.dart:347]: Move it to style

File: lib/pages/new_group/new_group_chat_info.dart [hunk 2]
```diff
@@ -320,3 +330,100 @@ class _AvatarForWebBuilder extends StatelessWidget {
...
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          const Padding(
+            padding: EdgeInsets.only(top: 8.0),
```
Comment (nqhhdev) [inline lib/pages/new_group/new_group_chat_info.dart:352]: same

File: lib/pages/new_group/new_group_chat_info.dart [hunk 3]
```diff
@@ -320,3 +330,100 @@ class _AvatarForWebBuilder extends StatelessWidget {
...
+          ),
+          const SizedBox(
+            width: 8.0,
+          ),
+          Expanded(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Padding(
+                  padding: const EdgeInsets.only(top: 8.0),
```
Comment (nqhhdev) [inline lib/pages/new_group/new_group_chat_info.dart:365]: same

File: lib/pages/new_group/new_group_chat_info.dart [hunk 4]
```diff
@@ -320,3 +330,100 @@ class _AvatarForWebBuilder extends StatelessWidget {
...
+                  child: Text(
+                    L10n.of(context)!.enableEncryption,
+                    style: Theme.of(context).textTheme.titleMedium?.copyWith(
+                          letterSpacing: 0.15,
```
Comment (nqhhdev) [inline lib/pages/new_group/new_group_chat_info.dart:369]: No need ``

File: lib/pages/new_group/new_group_chat_info.dart [hunk 5]
```diff
@@ -320,3 +330,100 @@ class _AvatarForWebBuilder extends StatelessWidget {
...
+                  builder: (context, isEnable, child) {
+                    return Column(
+                      children: [
+                        Text(
+                          L10n.of(context)!.encryptionMessage,
+                          style: Theme.of(context)
+                              .textTheme
+                              .bodyMedium
+                              ?.copyWith(
+                                letterSpacing: 0.4,
```
Comment (nqhhdev) [inline lib/pages/new_group/new_group_chat_info.dart:385]: same

File: lib/pages/new_group/new_group_chat_info.dart [hunk 6]
```diff
@@ -320,3 +330,100 @@ class _AvatarForWebBuilder extends StatelessWidget {
...
+                          alignment: Alignment.topCenter,
+                          duration: const Duration(milliseconds: 50),
+                          child: isEnable
+                              ? Text(
+                                  L10n.of(context)!.encryptionWarning,
+                                  style: Theme.of(context)
+                                      .textTheme
+                                      .bodyMedium
+                                      ?.copyWith(
+                                        letterSpacing: 0.4,
```
Comment (nqhhdev) [inline lib/pages/new_group/new_group_chat_info.dart:399]: same

File: lib/pages/new_group/widget/selected_participants_list.dart [hunk 7]
```diff
@@ -20,61 +24,130 @@ class _SelectedParticipantsListState extends State<SelectedParticipantsList> {
...
-                              ),
-                            ),
+              return const SizedBox.shrink();
+            }
+            return Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Padding(
+                  padding:
+                      const EdgeInsets.only(top: 12.0, left: 16.0, right: 16.0),
```
Comment (nqhhdev) [inline lib/pages/new_group/widget/selected_participants_list.dart:44]: move it to style

File: lib/pages/new_group/widget/selected_participants_list.dart [hunk 8]
```diff
@@ -20,61 +24,130 @@ class _SelectedParticipantsListState extends State<SelectedParticipantsList> {
...
+                            width: 1.0,
                           ),
                         ),
-                      )
-                      .toList(),
+                        labelPadding: const EdgeInsets.only(
+                          left: 8.0,
+                          top: 2.0,
+                          bottom: 2.0,
+                        ),
```
Comment (nqhhdev) [inline lib/pages/new_group/widget/selected_participants_list.dart:60]: same

File: lib/pages/new_group/widget/selected_participants_list.dart [hunk 9]
```diff
@@ -20,61 +24,130 @@ class _SelectedParticipantsListState extends State<SelectedParticipantsList> {
...
+                        padding: const EdgeInsets.all(0),
+                        label: Text(
+                          contact.displayName ?? contact.matrixId ?? '',
+                          style:
+                              Theme.of(context).textTheme.labelLarge?.copyWith(
+                                    letterSpacing: 0.1,
```
Comment (nqhhdev) [inline lib/pages/new_group/widget/selected_participants_list.dart:66]: No need

File: lib/pages/new_group/widget/selected_participants_list.dart [hunk 10]
```diff
@@ -20,61 +24,130 @@ class _SelectedParticipantsListState extends State<SelectedParticipantsList> {
...
+                                    .client
+                                    .getProfileFromUserId(
+                                      contact.matrixId!,
+                                      getFromRooms: false,
+                                    ),
+                                builder: ((context, snapshot) {
+                                  return Avatar(
+                                    mxContent: snapshot.data?.avatarUrl,
+                                    name: contact.displayName,
+                                    size: 24.0,
```
Comment (nqhhdev) [inline lib/pages/new_group/widget/selected_participants_list.dart:84]: Move it to style

File: lib/pages/new_group/widget/selected_participants_list.dart [hunk 11]
```diff
@@ -20,61 +24,130 @@ class _SelectedParticipantsListState extends State<SelectedParticipantsList> {
...
+                                  );
+                                }),
+                              )
+                            : Avatar(
+                                name: contact.displayName,
+                                size: 24.0,
+                              ),
```
Comment (nqhhdev) [inline lib/pages/new_group/widget/selected_participants_list.dart:91]: same

File: lib/pages/new_group/widget/selected_participants_list.dart [hunk 12]
```diff
@@ -20,61 +24,130 @@ class _SelectedParticipantsListState extends State<SelectedParticipantsList> {
...
+                ),
+                const SizedBox(
+                  height: 4.0,
+                ),
+                Padding(
+                  padding: const EdgeInsets.only(
+                    left: 16.0,
+                    right: 16.0,
+                    bottom: 8.0,
+                  ),
```
Comment (nqhhdev) [inline lib/pages/new_group/widget/selected_participants_list.dart:115]: same

File: lib/pages/new_group/widget/selected_participants_list.dart [hunk 13]
```diff
@@ -20,61 +24,130 @@ class _SelectedParticipantsListState extends State<SelectedParticipantsList> {
...
+                  child: Row(
+                    children: [
+                      Text(
+                        '${L10n.of(context)!.selectedUsers}${contactsNotifier.contactsList.length}',
```
Comment (nqhhdev) [inline lib/pages/new_group/widget/selected_participants_list.dart:119]: Update it

File: assets/l10n/intl_en.arb [hunk 14]
```diff
@@ -2746,5 +2746,9 @@
     "pasteImageFailed": "Paste image failed",
     "copyImageFailed": "Copy image failed",
     "fileFormatNotSupported": "File format not supported",
-    "copyImageSuccess": "Image copied to clipboard"
+    "copyImageSuccess": "Image copied to clipboard",
+    "encryptionMessage": "This feature protects your messages from being read by others, but also prevents them from being backed up on our servers. You can't disable this later.",
+    "encryptionWarning": "You might lose your messages if you access Twake app on the another device.",
+    "selectedUsers": "Selected users: ",
```
Comment (imGok) [inline assets/l10n/intl_en.arb:2752]: should add `:` in the view imo not in this string
```suggestion
    "selectedUsers": "Selected users",
```

**Review by hoangdat — APPROVED** [review]

File: lib/pages/new_group/new_group_chat_info.dart (hunk 5 above)
Comment (sherlockvn) [reply lib/pages/new_group/new_group_chat_info.dart:385]: Why ?

**Review by nqhhdev — APPROVED** [review]

**Pull request merged**